public/javascripts/ng-apps.js

```diff
@@ -41,6 +41,23 @@ springInitiative.config(function($stateProvider, $urlRouterProvider, $locationPr
         'content': {
           templateUrl: 'views/content/student.html',
           controller: 'studentController'
+        },
+        'studentView@index.student': {
+          templateUrl: 'views/content/studentView.html'
+        }
+      }
+    })
+    .state('index.student.addEntry', {
```

Comment:
https://github.com/angular-ui/ui-router/wiki/Nested-States-and-Nested-Views
hmm not exactly - there aren't any official examples of double nested views
i suppose you could also just do `child.grandchild:{parent: 'parent', templateURL...}`
the name just feels very long and unnecessary, but we can stay with it